packages/vaadin-crud/src/vaadin-crud.js

```diff
@@ -663,10 +664,6 @@ class CrudElement extends ElementMixin(ThemableMixin(PolymerElement)) {
     if (this.editedItem) {
       this.__onItemChange(this.editedItem);
     }
-    if (this.$.form != form) {
-      // move slotted from this to the slot parent
-      this.$.editor.appendChild(form);
-    }
```

Comment:
This change is no longer needed with the new approach. Previously, it was used here to make sure `this.childNodes` includes the form when called from the `vaadin-dialog-layout` component.

Note, the form is still moved to `vaadin-dialog-layout` - see comment below explaining that.